apps/docs/content/docs/sync.mdx

```diff
@@ -54,154 +55,153 @@ function MyApp() {
-production ready, but you can use it as a reference implementation to build it into your own
-existing backend.
+#### WebSocket server
 
-Read the reference docs for [`TLSocketRoom`](?) for more.
+The `@tldraw/sync-core` package exports a class called [`TLSocketRoom`](?) that should be created server-side on a per-document basis.
 
-### User presence
+`TLSocketRoom` is used to
 
-tldraw has support for displaying the 'presence' of other users. Presence information consists of:
+- Store an authoritative in-memory copy of the document state
+- Transparently set up communication between multiple sync clients via WebSockets.
+- Provide hooks for persisting the document state when it changes.
 
-- The user's pointer position
-- The user's set of selected shapes
-- The user's viewport bounds (the part of the canvas they are currently viewing)
-- The user's name, id, and a color to represent them
+> :bulb: You should make sure that there's only ever one `TLSocketRoom` globally for each room in your app. If there's
+> more than one, users won't see each other and will overwrite others' changes. We use [Durable
+> Objects](https://developers.cloudflare.com/durable-objects/) to achieve this on tldraw.com.
 
-This information will usually come from two sources:
+Read the reference docs for [`TLSocketRoom`](?), and see an example of how to use it in the [simple server example](https://github.com/tldraw/tldraw/blob/main/apps/simple-server-example/src/server/rooms.ts).
 
-- The tldraw editor state (e.g. pointer position, selected shapes)
-- The data layer of whichever app tldraw has been embedded in (e.g. user name, user id)
+#### Asset storage
 
-Tldraw is agnostic about how this data is shared among users. However, in order for tldraw to use the presence data it needs to be put into the editor's store as `instance_presence` records.
+As well as synchronizing the rapidly-changing document data, tldraw also needs a way to store and
+retrieve large binary assets like images or videos.
 
-We provide a helper for constructing a reactive signal for an `instance_presence` record locally, which can then be sent to other clients somehow. It is called [createPresenceStateDerivation](?).
+You'll need to make sure your backend can handle asset uploads & downloads, then implement
+[`TLAssetStore`](?) to connect it to tldraw.
 
-```ts
-import { createPresenceStateDerivation, react, atom } from 'tldraw'
+- Read about [how assets work in tldraw](/docs/assets).
+- Read the [`TLAssetStore`](?) reference docs.
+- See a complete example of an asset store in the
+  [`tldraw-sync-cloudflare`](https://github.com/tldraw/tldraw/blob/main/templates/sync-cloudflare/client/multiplayerAssetStore.tsx)
+  template.
 
-// First you need to create a Signal containing the basic user details: id, name, and color
-const user = atom<{ id: string; color: string; name: string }>('user', {
-	id: myUser.id,
-	color: myUser.color,
-	name: myUser.name,
-})
+#### Unfurling service
```

Comment:
I think we should avoid 'unfurl' in docs. I don't think it's very clear. What about "Bookmark preview service" or "URL metadata service" or something?

Reply:
'unfurl' seems to be pretty standard industry jargon? I'm fine with `URL metadata service` though. `Bookmark preview service` is a bit too specific or something.